services/webnn/ort/graph_builder_ort.cc

```diff
@@ -505,15 +506,169 @@ void GraphBuilderOrt::FindBoolOperands() {
+        const auto& cumulative_sum = *operation->get_cumulative_sum();
+        uint8_input_operands.insert(cumulative_sum.input_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kDequantizeLinear: {
+        const auto& dequantize_linear = *operation->get_dequantize_linear();
+        uint8_input_operands.insert(dequantize_linear.input_operand_id);
+        uint8_input_operands.insert(dequantize_linear.zero_point_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kElementWiseBinary: {
+        const auto& binary = *operation->get_element_wise_binary();
+        if (binary.kind == mojom::ElementWiseBinary::Kind::kAdd ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kSub ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kMul ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kDiv ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kMax ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kMin ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kEqual ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kGreater ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kGreaterOrEqual ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kLesser ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kLesserOrEqual ||
+            binary.kind == mojom::ElementWiseBinary::Kind::kNotEqual) {
+          uint8_input_operands.insert(binary.lhs_operand_id);
+          uint8_input_operands.insert(binary.rhs_operand_id);
+        }
+        break;
+      }
+      case mojom::Operation::Tag::kElementWiseUnary: {
+        const auto& unary = *operation->get_element_wise_unary();
+        if (unary.kind == mojom::ElementWiseUnary::Kind::kAbs ||
+            unary.kind == mojom::ElementWiseUnary::Kind::kErf ||
+            unary.kind == mojom::ElementWiseUnary::Kind::kIdentity) {
+          uint8_input_operands.insert(unary.input_operand_id);
+        }
+        break;
+      }
+      case mojom::Operation::Tag::kExpand: {
+        const auto& expand = *operation->get_expand();
+        uint8_input_operands.insert(expand.input_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kGather: {
+        const auto& gather = *operation->get_gather();
+        uint8_input_operands.insert(gather.input_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kGatherElements: {
+        const auto& gather_elements = *operation->get_gather_elements();
+        uint8_input_operands.insert(gather_elements.input_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kGatherNd: {
+        const auto& gather_nd = *operation->get_gather_nd();
+        uint8_input_operands.insert(gather_nd.input_operand_id);
+        break;
+      }
+      case mojom::Operation::Tag::kPad: {
+        // https://github.com/shiyi9801/chromium/issues/85
```

Comment:
Expand this comment into a TODO:
```suggestion
        // TODO(https://github.com/shiyi9801/chromium/issues/85): handle uint8 input once pad supports uint8 input.
```

Reply:
Done.